src/shared/components/Pagination/Pagination.style.tsx

```diff
@@ -1,50 +1,67 @@
-import { colors, sizes, transitions, typography } from '@/shared/theme'
+import { breakpoints, colors, sizes, transitions, typography } from '@/shared/theme'
 import { css } from '@emotion/react'
 import styled from '@emotion/styled'
+import Button from '../Button'
 
 type PaginationButtonProps = {
-  isChevron?: boolean
   isActive?: boolean
   isHidden?: boolean
 }
 
 export const PaginationWrapper = styled.div`
   display: flex;
+  max-width: 400px;
+  justify-content: space-between;
+  @media screen and (min-width: ${breakpoints.small}) {
+    max-width: unset;
+    justify-content: unset;
+  }
 `
 
 const buttonActiveState = css`
   color: ${colors.white};
   background-color: ${colors.gray[800]};
 `
 
+export const ChevronButton = styled(Button)`
+  width: ${sizes(12)};
+  height: ${sizes(12)};
+  @media screen and (min-width: ${breakpoints.small}) {
+    &:first-of-type {
+      margin-right: ${sizes(8)};
+    }
+    &:last-of-type {
+      margin-left: ${sizes(8)};
+    }
+  }
+`
+
 export const PaginationButton = styled.button<PaginationButtonProps>`
```

Comment:
The text color seems to be white when item is active, should be `gray[50]`

<img width="349" alt="Screenshot 2021-04-08 at 11 05 03" src="https://user-images.githubusercontent.com/12646744/113999563-70178400-985a-11eb-9662-e8a9f4e814b5.png">